doc/help/Messaging.md

```diff
@@ -35,3 +36,12 @@ Which sends a special mention notification to **alice** to check your message.
 To mention a teammate, press `@` and you should see a list of team members who can be messaged. You can either type their username or use the **Up** and **Down** arrow keys and then **ENTER** to select them to be mentioned. 
 
 You can configure how you'd like to be alerted about mentions of your username, your first name, your nickname, or other keywords from **Account Settings** > **Notifications** and you can set channel-specific preferences from **[Channel Name]** > **Notification Preferences** 
+
+## Messages Dropdown Menu
+
+The Messages Dropdown Menu contains additional actions you can perform on a message: 
```

Comment:
I might explain how you get to this menu. ie hover over the message, then click the three dot menu